lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -2783,9 +2783,14 @@ class ConvertAtenOnesOp : public OpConversionPattern<AtenOnesOp> {
     if (!adaptor.layout().getType().isa<Torch::NoneType>())
       return rewriter.notifyMatchFailure(op,
                                          "only default layout is supported");
+
     bool pinMemory;
     if (!adaptor.pin_memory().getType().isa<Torch::NoneType>() &&
```

Comment:
Since we are here already, it's safer and more readable to change `adaptor.pin_memory()` to `op.pin_memory()` as well. It's just a good practice to always check the original operands against torch types.

Reply:
Makes sense, thanks!